x-pack/plugins/security_solution/public/detection_engine/rule_details_ui/pages/rule_details/index.tsx

```diff
@@ -837,22 +817,14 @@ const RuleDetailsPageComponent: React.FC<DetectionEngineComponentProps> = ({
                       <EuiSpacer />
                     </Display>
                     {ruleId != null && (
-                      <AlertsTable
-                        filterGroup={filterGroup}
+                      <AlertsTableComponent
+                        configId={`${APP_ID}-rule-details`}
+                        flyoutSize="m"
+                        inputFilters={alertMergedFilters}
                         tableId={TableId.alertsOnRuleDetailsPage}
-                        defaultFilters={alertsTableDefaultFilters}
-                        hasIndexWrite={hasIndexWrite ?? false}
-                        hasIndexMaintenance={hasIndexMaintenance ?? false}
                         from={from}
-                        loading={loading}
-                        showBuildingBlockAlerts={showBuildingBlockAlerts}
-                        showOnlyThreatIndicatorAlerts={showOnlyThreatIndicatorAlerts}
-                        onShowBuildingBlockAlertsChanged={onShowBuildingBlockAlertsChangedCallback}
-                        onShowOnlyThreatIndicatorAlertsChanged={
-                          onShowOnlyThreatIndicatorAlertsCallback
-                        }
-                        onRuleChange={refreshRule}
                         to={to}
+                        isLoading={showUpdating}
```

Comment:
Nice! This looks like a substantial simplification of the interface!

Reply:
Yes, it looks like a simplification but the logic is distributed at 2 places. New Alert Table follows registry pattern, where we register some props when the app starts. Those props will not change during the APP run.

You can see the `registry` here : https://github.com/elastic/kibana/pull/149128/files#diff-c4f8812aff61aabd31bc393385a26001c49c132946a5f8a1d17ba309ef421332